package.json

```diff
@@ -171,6 +171,7 @@
     "@commitlint/cli": "latest",
     "@commitlint/config-conventional": "latest",
     "husky": "latest",
+    "typescript-eslint": "^0.0.1-alpha.0",
```

Comment:
Why is this added to the Dependencies?

Reply:
That is indeed a mistake, I am still trying to make pre-commit.ci happy....